monkey/common/common_consts/post_breach_consts.py

```diff
@@ -9,3 +9,4 @@
 POST_BREACH_SIGNED_SCRIPT_PROXY_EXEC = "Signed script proxy execution"
 POST_BREACH_ACCOUNT_DISCOVERY = "Account discovery"
 POST_BREACH_CLEAR_CMD_HISTORY = "Clear command history"
+POST_BREACH_PROCESS_LIST_COLLECTION = "Process list collection"
```

Comment:
Consider renaming like an action
```suggestion
POST_BREACH_PROCESS_LIST_COLLECTION = "Collect running processes"
```